### DISCUSSION TITLE
AdGuard Home repo: Broken image

### DISCUSSION THREAD

**Discussion by ghost** [community/community#46031] [Mobile] [answered]
<details>
<summary>ℹ️ Additional Debug Info</summary>

https://github.com/AdguardTeam/AdGuardHome

Mobile app
![image](https://user-images.githubusercontent.com/95468948/216115095-6bc5a16b-c102-4a2f-b47d-7095a672a9db.png)

Web page![image](https://user-images.githubusercontent.com/95468948/216114935-fa4bc079-7faa-4a39-b42d-d34d02fda663.png)

GitHub for iOS, Version 1.98.0 (90123676)
Device: iPhone 13 Pro (iOS 16.3)
</details>

**Comment by alcere** [comment]
@jslawler thanks for reporting, I have created an issue and the team will look at it as soon as possible.

**Reply by ghost** [reply, marked as answer]
Resolved in latest update.
